server/templates/staff/course/section/section.list.html

```diff
@@ -0,0 +1,104 @@
+<div class="box">
+    <div class="box-header">
+        <h3 class="box-title"><span> {{ title }} </span></h3>
+        <div class="box-tools">
+            <div class="box-tools pull-right">
+
+                <button type="button" class="btn btn-box-tool" data-widget="collapse" data-toggle="tooltip" title="Collapse">
+                    <i class="fa fa-minus"></i>
+                </button>
+
+            </div>
+        </div>
+    </div>
+    <div id="emails" style="display: none;">{{emails}}</div>
+    <script>
+    function copyToClipboard() {
+        var text = $("#emails").text();
+        if (window.clipboardData && window.clipboardData.setData) {
+            return clipboardData.setData("Text", text);
+        } else if (document.queryCommandSupported && document.queryCommandSupported("copy")) {
+            var textarea = document.createElement("textarea");
+            textarea.textContent = text;
+            textarea.style.position = "fixed";  // Prevent scrolling to bottom of page in MS Edge.
+            document.body.appendChild(textarea);
+            textarea.select();
+            try {
+                return document.execCommand("copy");  // Security exception may be thrown by some browsers.
+            } catch (ex) {
+                console.warn("Copy to clipboard failed.", ex);
+                return false;
+            } finally {
+                document.body.removeChild(textarea);
+                alert("Emails copied to clipboard! Right click and paste anywhere.");
+            }
+        }
+    }
+</script>
+
+    <!-- /.box-header -->
+    <div class="box-body table-responsive no-padding table-loading" style="display: block;" id="{{ title }}-list">
+        <table class="table table-hover">
+            <thead>
+                <th class="sort" data-sort="email">User</th>
+                <th class="sort" data-sort="name">Name</th>
+                <th class="sort" data-sort="sid">SID</th>
+                <th class="sort sort-padding" data-sort="secondary">Course ID</th>
+                <th class="sort" data-sort="role">Role</th>
+                <th class="sort sort-padding" data-sort="timestamp">Enrolled At</th>
+                <th class="sort sort-padding" data-sort="section">Section</th>
+            </thead>
+
+            <tbody class="list">
+                {%- for item in roster %}
```

Comment:
This should be pretty easy to just add a web driver test for this logic and make sure that all students assigned to a section page when visited show up.